lntest/itest/lnd_taproot_test.go

```diff
@@ -906,7 +906,14 @@ func testTaprootMuSig2CombinedLeafKeySpend(ctxt context.Context, t *harnessTest,
 		ctxt, &signrpc.MuSig2SignRequest{
 			SessionId:     sessResp3.SessionId,
 			MessageDigest: sigHash,
-			Cleanup:       true,
+		},
+	)
+	require.NoError(t.t, err)
+
+	// We manually clean up session 3, just to make sure that works as well.
+	_, err = alice.SignerClient.MuSig2Cleanup(
+		ctxt, &signrpc.MuSig2CleanupRequest{
+			SessionId: sessResp3.SessionId,
```

Comment:
Add an assertion that trying to do another signature w/ the same session ID fails?